ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/project/ProjectServiceClientImpl.java

```diff
@@ -242,6 +242,12 @@ public void apply(PromiseError arg) throws OperationException {
 
     /** {@inheritDoc} */
     @Override
+    public String getFileContentUrl(Path path) {
```

Comment:
i propose you localize this changes here https://github.com/eclipse/che/blob/master/ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/imageviewer/ImageViewer.java#L153-153